WordPress/Classes/ViewRelated/Notifications/ReplyTextView/ReplyTextView.swift

```diff
@@ -299,7 +301,18 @@ import WordPressShared.WPStyleGuide
 
     fileprivate func refreshReplyButton() {
         let whitespaceCharSet = CharacterSet.whitespacesAndNewlines
-        replyButton.isEnabled = textView.text.trimmingCharacters(in: whitespaceCharSet).isEmpty == false
+        let isEnabled = self.textView.text.trimmingCharacters(in: whitespaceCharSet).isEmpty == false
+
+        if isEnabled == self.replyButton.isEnabled {
+            return
+        }
```

Comment:
I guess we can move this logic of checking if we should refresh the button or not to a `private` method with a meaningful name. What do you think?

Reply:
I agree.